showcase/src/app/app.component.html

```diff
@@ -1,29 +1,68 @@
-<header class="stark-header aah-stark">
+<header class="stark-app-header">
 	<div class="stark-container">
 		<div class="stark-app-bar">
-			<button color="white" mat-icon-button>
-				<mat-icon starkSvgViewBox svgIcon="home"></mat-icon>
-			</button>
-			<button color="white" mat-icon-button (click)="sidenav.toggle()">
-				<mat-icon starkSvgViewBox svgIcon="menu"></mat-icon>
-			</button>
-			<button color="white" mat-icon-button>
-				<mat-icon starkSvgViewBox svgIcon="arrow-left"></mat-icon>
-			</button>
+			<div class="stark-content-left">
```

Comment:
Maybe this class could be renamed to `stark-app-bar-content-left` to be more descriptive.